objects.js

```diff
@@ -8520,6 +8523,48 @@ WatcherMorph.prototype.userMenu = function () {
     return menu;
 };
 
+WatcherMorph.prototype.valueExporter = function (format) {
+    var myself = this,
+        value = this.currentValue,
+        contents,
+        format = format || 'plain',
+        ide = myself.parentThatIsA(IDE_Morph);
+
+    return function () {
+        switch (format) {
+            case 'plain':
+                contents = value instanceof List ? 
+                    value.asArray().join(', ') : 
```

Comment:
Ah, yes, the delimiter debate. :)
 my vote here is for `\n` (since that's what BYOB 3 did) and it makes `split (data) by [line]` really easy to do. (Whereas with a `, ` delimiter then students will likely run into issues with `,` vs `, ` (which I mean `,\s`, since that's probably hard to tell the difference).